rental.rb

```diff
@@ -0,0 +1,11 @@
+require './book'
+require './person'
+
+class Rental
+  attr_accessor :date, :book, :person
+
+  def initialize(date, person)
+    @date = date
+    @person = person
+  end
```

Comment:
- [ ] Please try to add more explicit relations like A rental belongs to a book and also belongs to a person:

you can add those snippets in initialize method of rental class to implement that relationships:

````ruby
 # *The rental is adding itself to the @book achieving the 'belongs-to' association
    book.rentals << self
````
````ruby
    # *The rental is adding itself to the @person the 'belongs-to' association
    person.rentals << self
````

Reply:
sorry but I don't understand this, I add this into the class and create the methods on it?

Reply:
ok, I added this relation to the rental.rb file